resources/synapse_admin.rb

```diff
@@ -0,0 +1,36 @@
+# Synapse Admin Web Application
+# This resource allows for the deployment of an Admin web-interface for Matrix Synapse servers.
+# NOTE: Requires /_synapse/admin to be reachable from the browser
+
+resource_name :osl_synapse_admin
+provides :osl_synapse_admin
+unified_mode true
+
+default_action :create
+
+property :matrix_domain, String, name_property: true
```

Comment:
So in theory this supports multiple URL's according to the documentation. We should make sure this supports both Strings or an Array.  Also I think this name might be better.
```suggestion
property :home_server, String, name_property: true
```